Core/Src/main.c

```diff
@@ -112,46 +110,37 @@ int main(void)
   MX_TIM17_Init();
   /* USER CODE BEGIN 2 */
   BMP_Init();
-  HMC_DevId();
-  HMC_Set_SingleMeasureMode();
+  HMC_Init();
   ADXL_Init();
-  GYRO_PowerMode();
-
-  int16_t x,y,z;
-
-  ADXL_ReadValuesXYZ(&x, &y, &z);
-  GYRO_ReadValuesXYZ(&x, &y, &z);
+  GYRO_Init();
   /* USER CODE END 2 */
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
-
-	while (1)
+    while (1)
 	{
 		if (ADXL_IsExtiDataReady())
 		{
-			ADXL_IT_StartSPI();
+			ADXL_StartMeasurement();
 		}
 
 		if (GYRO_IsExtiDataReady())
 		{
-			GYRO_SPI_IT_START();
+			GYRO_StartMeasurement();
 		}
 
 		if (is_new_ACC_datacomplete)
 		{
 			is_new_ACC_datacomplete = false;
-			ADXL_IT_GetValuesXYZ(&x, &y, &z);
-			ADXL_ConvertXYZValuesG(&x, &y, &z);
-			USART_TransmitACCValues(&x, &y, &z);
+			ADXL_GetValues(&x, &y, &z);
+		    USART_TransmitACCValues(&x, &y, &z);
```

Comment:
use a consistent number of spaces for tabs.
Here it looks like it is set to 8.
Everywhere else it is set to 4.